scripting/pugsetup.sp

```diff
@@ -323,6 +325,10 @@ public void OnPluginStart() {
   g_WarmupMoneyOnSpawnCvar = CreateConVar(
       "sm_pugsetup_money_on_warmup_spawn", "1",
       "Whether clients recieve 16,000 dollars when they spawn. It's recommended you use mp_death_drop_gun 0 in your warmup config if you use this.");
+      
+  g_EndGameOnLastPlayerInServerCvar = CreateConVar(
+  "sm_pugsetup_forceend_on_players_leave", "0", 
+  "Should PugSetup end match if is just 1 player in match and it is LIVE? (its good enable if you have server hibernation actives, it will prevens crash.)");
 
```

Comment:
No need to mention the plugin name in a cvar description, or put "LIVE" in all caps here.

"Whether to force end a live match if only 1 player is left in the server. This may help prevent crashes for servers with hibernation enabled."

Reply:
Done ^^